<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AcEquipEditorClass</class>
 <widget class="QWidget" name="AcEquipEditorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1183</width>
    <height>1015</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>AcEquipEditor</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_52" stretch="0,1,0,0">
     <item>
      <widget class="QLabel" name="label_263">
       <property name="text">
        <string>Quest List</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="listWidgetQuests"/>
     </item>
     <item>
      <widget class="QPushButton" name="buttonAddNewQuest">
       <property name="text">
        <string>Add New</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_53" stretch="0,1">
     <item>
      <widget class="QLabel" name="label_264">
       <property name="text">
        <string>Quest Data</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame">
       <property name="frameShape">
        <enum>QFrame::Shape::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Shadow::Plain</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_201" stretch="2,1,2,1">
          <item>
           <widget class="QSpinBox" name="spinBoxQuestId">
            <property name="maximum">
             <number>9999999</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_265">
            <property name="text">
             <string>Quest ID</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBoxPlayerCount">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>4</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_266">
            <property name="text">
             <string>Player Count</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_205" stretch="2,1,2,1">
          <item>
           <widget class="QSpinBox" name="timeEditRankA">
            <property name="suffix">
             <string>s</string>
            </property>
            <property name="maximum">
             <number>99999999</number>
            </property>
            <property name="singleStep">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_273">
            <property name="text">
             <string>A Rank Time</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBoxRankAPoints">
            <property name="maximum">
             <number>999999999</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_274">
            <property name="text">
             <string>A Rank Points</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_206" stretch="2,1,2,1">
          <item>
           <widget class="QSpinBox" name="timeEditRankB">
            <property name="suffix">
             <string>s</string>
            </property>
            <property name="prefix">
             <string/>
            </property>
            <property name="maximum">
             <number>99999999</number>
            </property>
            <property name="singleStep">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_275">
            <property name="text">
             <string>B Rank Time</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBoxRankBPoints">
            <property name="maximum">
             <number>999999999</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_276">
            <property name="text">
             <string>B Rank Points</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_207" stretch="2,1,2,1">
          <item>
           <widget class="QSpinBox" name="timeEditRankC">
            <property name="suffix">
             <string>s</string>
            </property>
            <property name="prefix">
             <string/>
            </property>
            <property name="maximum">
             <number>99999999</number>
            </property>
            <property name="singleStep">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_277">
            <property name="text">
             <string>C Rank Time</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBoxRankCPoints">
            <property name="maximum">
             <number>999999999</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_278">
            <property name="text">
             <string>C Rank Points</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTabWidget" name="tabWidgetEquipSets">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tab">
           <attribute name="title">
            <string>Tab 1</string>
           </attribute>
          </widget>
          <widget class="QWidget" name="tab_2">
           <attribute name="title">
            <string>Tab 2</string>
           </attribute>
          </widget>
          <widget class="QWidget" name="tab_3">
           <attribute name="title">
            <string>Page</string>
           </attribute>
          </widget>
          <widget class="QWidget" name="tab_5">
           <attribute name="title">
            <string>Page</string>
           </attribute>
          </widget>
          <widget class="QWidget" name="tab_4">
           <attribute name="title">
            <string>Page</string>
           </attribute>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
